### PR TITLE
Refs #23190 - replace .uniq with .distinct

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: app/models/katello/content_view.rb [hunk 1]
```diff
@@ -353,16 +353,6 @@ def products(env = nil)
       Product.joins(:repositories).where("#{Katello::Repository.table_name}.id" => repos.map(&:id)).distinct
     end
 
-    def version_products(env)
```
Comment (jturel) [inline app/models/katello/content_view.rb]: dead code I guess?

Comment (jlsherrill) [reply app/models/katello/content_view.rb]: Yep!

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
[test katello]

**Pull request merged**